source/modules/assets/softwares.rst

```diff
@@ -0,0 +1,147 @@
+Softwares
+=========
+
+GLPI allows management of softwares and their versions as well as licenses, associated or not to software versions.
+
+A software is by default associated with an entity.
+
+.. ??? Un logiciel est par défaut associé à une entité : c'est-à-dire qu'il y aura autant de logiciels dans la base qu'il y a d'entités. Rendre un logiciel visible dans les sous-entités permet de faire une gestion plus fine.
```

Comment:
Todo I guess. The part "c'est-à-dire qu'il y aura autant de logiciels dans la base qu'il y a d'entités." => sounds wrong. ping @orthagh

Reply:
the wording is indeed strange.

The feature described here is the entity usage when importing a software (with an automatic inventory)
Do we create the new software in the asset entity or in a upper entity (usually root entity) and create software_version in the asset entity.

Doing the first way duplicates same software for each entity (when importing assets in different entities)

![image](https://user-images.githubusercontent.com/418844/104316501-d8b1c780-54dc-11eb-9907-c1a00479734a.png)

Reply:
In fusion plugin, software entitiy id is retrieved from entities configuration (default to 0), and item_software entity id is the one of the parent asset

Reply:
I propose to just drop; I'm not able to provide any comprehensive contents (without writing at least an entire paragraph). Sounds of of the scope of the current PR.